core/common/cluster.go

```diff
@@ -54,6 +54,8 @@ const (
 	ClusterStatusFreed    = "Freed"
 	ClusterStatusDeleting = "Deleting"
 	ClusterStatusCreating = "Creating"
+	// ClusterStatusMaintaining means no publishing operation is allowed on the cluster currently
+	ClusterStatusMaintaining = "Maintaining"
```

Comment:
why not use `UnderMaintenance` as the status? (Just a Suggestion)

Reply:
Same reason, consistence with 'Deleting', 'Creating', etc